### ISSUE TITLE
`isinstance` should reject use of `Annotated` type alias

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#7092]
```python
X = Annotated[int, ""]

print(isinstance(1, X))
```

The above code generates a runtime error, but pyright doesn't detect it.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.349, which I just published.
